framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/OpenShiftOperatorScenario.java

```diff
@@ -108,6 +118,25 @@ private void createRoleBindingsInProject(Project project) {
         project.getOpenShiftAdmin().rbac().roleBindings().inNamespace(project.getName()).create(roleBinding);
     }
 
+    private void createClusterRoleInProject(Project project) {
+        logger.info("Creating cluster role in project '" + project.getName() + "' from " + OpenShiftResource.CLUSTER_ROLE.getResourceUrl().toString());
+        ClusterRole role = project.getOpenShiftAdmin().rbac().clusterRoles().load(OpenShiftResource.CLUSTER_ROLE.getResourceUrl()).get();
+        project.getOpenShiftAdmin().rbac().clusterRoles().inNamespace(project.getName()).createOrReplace(role);
+    }
+
+    private void createClusterRoleBindingsInProject(Project project) {
+        logger.info("Creating cluster role bindings in project '" + project.getName() + "' from " + OpenShiftResource.CLUSTER_ROLE_BINDING.getResourceUrl().toString());
+        ClusterRoleBinding roleBinding = project.getOpenShiftAdmin().rbac().clusterRoleBindings().load(OpenShiftResource.CLUSTER_ROLE_BINDING.getResourceUrl()).get();
+        roleBinding.getMetadata().setName(OPERATOR_DEPLOYMENT_NAME + project.getName());
+        roleBinding.getMetadata().setNamespace(project.getName());
+        roleBinding.getSubjects().forEach(subject -> subject.setNamespace(project.getName()));
```

Comment:
we can use projectName or getNamespace() instead of project.getName
```suggestion
        roleBinding.getMetadata().setName(OPERATOR_DEPLOYMENT_NAME + projectName);
        roleBinding.getMetadata().setNamespace(projectName);
        roleBinding.getSubjects().forEach(subject -> subject.setNamespace(projectName));
```

Reply:
Ok. Will use getNamespace() method.